### PR TITLE
add continental shelf data layers

### PR DESCRIPTION
**Pull request by hsimonoko**
## Description

add continental shelf (north, northeast, south) points, lines, and polygons layers

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: qgreenland/config/datasets/continental_shelf.py
Comment (MattF-NSIDC) [inline qgreenland/config/datasets/continental_shelf.py:66]: TODO:

- [ ] Fill out a dataset abstract
- [ ] Citation text
- [ ] Citation URL

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]

File: qgreenland/config/datasets/continental_shelf.py
Comment (MattF-NSIDC) [inline qgreenland/config/datasets/continental_shelf.py:64]: TODO: 

- [ ] Dataset title: instead of the link text, we should use a title that's searchable and maybe related to a publication or oganization. "Continental Shelf Programme"? Maybe we can build an abstract and/or citation from the stuff here as well:

http://continentalshelf.org/about.aspx

File: qgreenland/config/helpers/layers/continental_shelf.py
Comment (MattF-NSIDC) [inline qgreenland/config/helpers/layers/continental_shelf.py:11]: TODO:

- [ ] For each description, more clearly describe what the user is looking at. e.g. "Points representing the boundary of the Northern extended continental shelf of Greenland."

Twila asked about the polygons -- what does the area within the polygon mean? Maybe we should remove the polygons if the area within the polygon is unimportant. We should probably follow-up with Vinton. We shouldn't wait for a response from him before merging this, just planning ahead. Let's do a first draft on a better description before merge though.

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]

**Comment by MattF-NSIDC** [comment]
Discussion in Slack: https://nsidc.slack.com/archives/CSXBA3ERE/p1639171808084300

**Comment by MattF-NSIDC** [comment]
Let's also include a note in the "description" for the polygons that the inside edge is the EEZ.

TODO:

- [ ] Update polygon descriptions to note the meaning of the inside edges

File: qgreenland/config/helpers/layers/continental_shelf.py
Comment (trey-stafford) [inline qgreenland/config/helpers/layers/continental_shelf.py:86]: Lets use the `decompress` helper for this step, `qgreenland.config.helpers.steps.decompress`.

File: qgreenland/config/helpers/layers/continental_shelf.py
Comment (trey-stafford) [inline qgreenland/config/helpers/layers/continental_shelf.py:94]: Looks like we can use the `ogr2ogr` helper here, `qgreenland.config.helpers.steps.ogr2ogr`.

Comment (trey-stafford) [reply qgreenland/config/helpers/layers/continental_shelf.py:94]: Nevermind, the default `ogr2ogr` helper requires a clip boundary. We don't want to clip this layer.

File: qgreenland/config/layers/Environmental management/Maritime boundaries/__settings__.py
Comment (trey-stafford) [inline qgreenland/config/layers/Environmental management/Maritime boundaries/__settings__.py:2]: Can we remove this commented-out import?

**Review by trey-stafford — CHANGES_REQUESTED** [review]
Just a couple of fixups requested!

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**
